Lib/idlelib/configdialog.py

```diff
@@ -21,6 +21,7 @@
 from tkinter import colorchooser
 import tkinter.font as tkfont
 from tkinter import messagebox
+from tkinter.simpledialog import _setup_dialog
```

Comment:
As explained on the issue, I don't want this dialog changed on mac.  I don't know what happens on Linux.

Reply:
Done.